### PR TITLE
Refactor Person to allow setting of userId in builder directly.

### LINKED ISSUE #17: Allow Persons to be created with a preset ID rather than a new, randomly generated one.
Currently, there is no way to create an instance of a Person without generating a new ID for them. This complicates the process of creating instances of said classes where the ID is known (for example, constructing a Person object from information stored in the database).

### PR DESCRIPTION
**Pull request by JosephBushagour**
Previously, it was impossible (well, very annoying) to set the ID of a person to a known ID during creation. This was a mistake, and is fixed with this PR.

### CODE REVIEW

**Review by KeatonTech — APPROVED** [review]

File: server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java [hunk 1]
```diff
@@ -67,9 +67,11 @@ public static Person getPerson(DatabaseClient dbClient, String userId) {
               Key.of(userId),
               Arrays.asList("email", "nickname", "pronouns"));
     if (row != null) {
-      Person.Builder personBuilder = Person.newBuilder(row.getString(/*emailIndex=*/0),
-                                                       row.getString(/*nicknameIndex=*/1));
-      // TODO: implement setting the userId field @JosephBushagour
+      Person.Builder personBuilder = Person.newBuilder()
+                                           .setEmail(row.getString(/*emailIndex=*/0))
```
Comment (SarahGB) [inline server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java]: There should be a spaces in this line: `/* emailIndex= */ 0`. Please update here and the line below.

File: server/src/test/java/com/google/coffeehouse/common/PersonTest.java [hunk 2]
```diff
@@ -39,20 +38,15 @@
   private static final String ALT_EMAIL = "New Email";
   private static final String PRONOUNS = "she/her";
   private static final String ALT_PRONOUNS = "New Pronouns";
-  private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
+  private static final String USERID = "predetermined-identification-string";
```
Comment (SarahGB) [inline server/src/test/java/com/google/coffeehouse/common/PersonTest.java]: Nit: As UserId is treated as two separate words below, consider changing this to USER_ID.

**Review by SarahGB — APPROVED** [review]

File: server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java (hunk 1 above)
Comment (JosephBushagour) [reply server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java]: Good catch! All parameters comments in this file were updated to fit these guidelines.

File: server/src/test/java/com/google/coffeehouse/common/PersonTest.java (hunk 2 above)
Comment (JosephBushagour) [reply server/src/test/java/com/google/coffeehouse/common/PersonTest.java]: Absolutely correct, I've switched all instances of `USERID` in this PR to `USER_ID`.

**Review by linamontes10-zz — APPROVED** [review]

File: server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java (hunk 1 above)
Comment (linamontes10-zz) [reply server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java]: This was my error -- sorry about that, and thank you for fixing that Joey!

**Pull request merged**